distribution/alliance-branding-app/src/main/resources/features.xml

```diff
@@ -1,28 +1,28 @@
-<!--
-/**
- * Copyright (c) Codice Foundation
- *
- * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
- * version 3 of the License, or any later version.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
- * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- *
- **/
--->
-<features name="${project.artifactId}-${project.version}"
-          xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
-
-    <feature name="alliance-branding" version="${project.version}"
-             description="Alliance Branding">
-        <feature prerequisite="true">web-container</feature>
-        <feature prerequisite="true">branding-api</feature>
-        <bundle>mvn:commons-io/commons-io/${commons-io.version}</bundle>
-        <bundle>mvn:commons-configuration/commons-configuration/${commons-configuration.version}</bundle>
-        <bundle>mvn:org.codice.alliance.distribution/alliance-branding-plugin/${project.version}</bundle>
-    </feature>
-
-</features>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<features xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="${project.artifactId}-${project.version}"
+          xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
+          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
+
+    <feature name="alliance-branding" version="${project.version}"
+             description="Alliance Branding" install="auto">
```

Comment:
You can remove the install="auto" here because of the changes to the application service that starts the feature with the same name? If so should this be named alliance-branding-app

Reply:
I think specifying one main feature works:
https://github.com/codice/ddf/blob/08d74f3a7d34dde689f45d577f7d11573d2ff400/platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/Application.java#L70-L79
https://github.com/codice/ddf/blob/6c339de56e0a9a8d5b75cf060bd5c488ccb99ee8/platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationImpl.java#L83-L107